### PR TITLE
Removed broken link

### PR DESCRIPTION
**Pull request by thecodingaviator**
Removed broken link from README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -273,7 +273,6 @@ Code Vectors: Understanding Programs Through Embedded Abstracted Symbolic Traces
 * [Syntax-Directed Variational Autoencoder for Structured Data](https://mlatgt.blog/2018/02/08/syntax-directed-variational-autoencoder-for-structured-data/)
 * [Weighted MinHash on GPU helps to find duplicate GitHub repositories.](https://blog.sourced.tech//post/minhashcuda/)
 * [Source Code Identifier Embeddings](https://blog.sourced.tech/post/id2vec/)
-* [Using recurrent neural networks to predict next tokens in the java solutions](http://near.ai/articles/2017-06-01-Code-Completion-Demo/)
```
Comment (EgorBu) [inline README.md]: Thank you! Could you put https://codeforces.com/blog/entry/52327 instead of a broken link?

**Review by EgorBu — CHANGES_REQUESTED** [review]
It's better to put another link instead of a broken one.

File: README.md (hunk 1 above)
Comment (vcoisne) [reply README.md]: cc @thecodingaviator ! Are you open to adding this new link ?

**Comment by thecodingaviator** [comment]
Yup, will do today

File: README.md (hunk 1 above)
Comment (thecodingaviator) [reply README.md]: @vcoisne Doing

Comment (thecodingaviator) [reply README.md]: @EgorBu Done

**Comment by thecodingaviator** [comment]
@EgorBu Done!

**Comment by m09** [comment]
Hi @thecodingaviator, thanks for the PR! We'd need a DCO before merging. You can get more infos about it [here](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md)

**Comment by thecodingaviator** [comment]
@m09 I'm not sure how to add DCO, do I have to pull my fork from GitHub, make a change, commit with the message - `Signed-off-by: Random J Developer <random@developer.example.org>` , push it to my fork and open a new PR?

**Comment by smola** [comment]
@thecodingaviator No need to create a new PR. You can update this one with sign-off as follows:

```
git clone git@github.com:thecodingaviator/awesome-machine-learning-on-source-code.git
cd awesome-machine-learning-on-source-code
git checkout patch-1
git commit --amend --no-edit --signoff
git push origin +patch-1
```

This will clone your repo, checkout the branch from the PR (`patch-1`), change the last commit to add the sign-off message, then push to the same branch overwriting it. When changing the branch `patch-1`, this PR will update automatically.

**Comment by thecodingaviator** [comment]
@smola Done

**Pull request merged**